lib/path/tests/generate.awk

```diff
@@ -28,10 +29,13 @@ BEGIN {
 
   # add extra weight for null, indicating the end of the string
   # Must be at least 1 to have strings end at all
-  total = upperslash + 1 + extranullweight
+  uppernewstring = upperslash + 1 + extranullweight
+
+  total = uppernewstring + 1 + extraelemweight
```

Comment:
I'm really not fond of this extra complexity, see the comment at the end of the Nix file.